OpenSim/Simulation/Model/ModelComponent.h

```diff
@@ -79,9 +78,18 @@ class ModelDisplayHints;
  *
  * @author Ajay Seth, Michael Sherman
  */
+
 class OSIMSIMULATION_API ModelComponent : public Component {
 OpenSim_DECLARE_ABSTRACT_OBJECT(ModelComponent, Component);
-
+// PROPERTIES
+//==============================================================================
+/** @name Property declarations
+These are the serializable properties associated with this class. **/
+/**@{**/
+public:
+    OpenSim_DECLARE_LIST_PROPERTY(GeometrySet, Geometry,
```

Comment:
I am confused by this property name. I thought it was an object but was unable to locate a `GeometrySet` class -- did I miss it? If it is not the name of an object, then it doesn't follow our convention for property names, which would be something like `geometry_set`.